torch/nn/functional.py

```diff
@@ -1379,6 +1379,17 @@ def linear(input, weight, bias=None):
 
 def bilinear(input1, input2, weight, bias=None):
     # type: (Tensor, Tensor, Tensor, Optional[Tensor]) -> Tensor
+    r"""
+    Applies a bilinear transformation to the incoming data: :math:`y = x_1 A x_2 + b`
+
+    Shape:
+
+        - input1: :math:`(N, *, H_{in1})` where :math:`H_{in1}=\text{in1\features}` and :math:`*` means any number of additional dimensions. All but the last dimension of the inputs should be the same.
```

Comment:
This line is triggering a `line too long` lint error.

Reply:
What is the maximum line length? `CONTRIBUTING.md` says 80 characters "inside docstrings block." `r"""` begins after four spaces on line 1382. Wouldn't this also mean that line 1383 is too long (86 characters, or 82 characters from beginning of docstrings block)? I will format my revisions according to this 80 character limit, but would like to know the guidelines for future reference. Thank you!

Reply:
Yeah I guess you're probably right that you should probably limit it to 80 characters if possible, so it can't hurt to also split line 1382. That said, two characters over might not be the biggest of issues, I don't know. Someone with more knowledge on the Jupyter documentation popups could pitch in.

What I will point out is that if you click the "Details" link next to the lint failure, you can see exactly what it's complaining about:

![image](https://user-images.githubusercontent.com/9757500/63616326-54903480-c5b5-11e9-9ab9-339f17a30050.png)

As you can see, it's somewhat permissive in that it allows you to have lines of up to 120 characters if you really need to, but that's for all lines (not just documentation), and you probably don't really need to go that long in most cases.
The main thing to note is that you should never have a lint failure because that will surely prevent your PR from being merged.
Hope that helps.

Reply:
Thank, this is really helpful.